admin_guide/manage_authorization_policy.adoc

```diff
@@ -0,0 +1,357 @@
+= Managing Authorization Policy
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can use link:../cli_reference/overview.html[the CLI] to view
+link:../architecture/additional_concepts/authorization.html[authorization
+policy] and the administrator CLI to manage the
+link:../architecture/additional_concepts/authorization.html[roles and bindings]
+within a policy.
+
+== Viewing Roles and Bindings
+link:../architecture/additional_concepts/authorization.html#roles[Roles] grant
+various levels of access in the system-wide
+link:../architecture/additional_concepts/authorization.html#cluster-policy-and-local-policy[cluster
+policy] and/or project-scoped
```

Comment:
I believe using and/or is frowned upon. Suggestion: "Roles grant various levels of access in the system-wide cluster policy _as well as_ project-scoped local policies"